unlock-app/src/__tests__/reducers/locksReducer.test.js

```diff
@@ -110,6 +111,23 @@ describe('locks reducer', () => {
     })
   })
 
+  it('should add the name and address of a lock when receiving SET_LOCK_NAME', () => {
+    const lockName = 'foo'
+    const reduction = reducer(
```

Comment:
nit - reducers return state in reducer, makes it slightly faster to read than translating reduction mentally